src/Pos/Client/CLI/Secrets.hs

```diff
@@ -14,11 +14,13 @@ import           Pos.Genesis         (genesisDevSecretKeys)
 import           Pos.Ssc.GodTossing  (genesisDevVssKeyPairs)
 import           Pos.Util.UserSecret (UserSecret, usPrimKey, usVss, writeUserSecret)
 
-import           NodeOptions         (Args (..))
+import           Pos.Client.CLI.NodeOptions         (SimpleNodeArgs (..))
 
+-- TODO (jk): Don't use Args, we do need just `devSpendingGenesisI`
+-- So we can use `userSecretWithGenesisKey` for simple + wallet node
```

Comment:
It seems like you already came up with a solution which doesn't require this, so this TODO is obsolete.